keyboards/naked48/rules.mk

```
@@ -0,0 +1,73 @@
+# SRC += i2c.c
+SRC += serial.c
```

This should be added to `QUANTUM_LIB_SRC` instead.  The reason for this, is that if somebody enables LTO, it can (will) mess with the serial communication.  Using the `QUANTUM_LIB_SRC` path prevents LTO from optimizing it, and causing the weird issues 

```suggestion
QUANTUM_LIB_SRC += serial.c
```